tests/src/Search/Workflow/WorkflowTestCase.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace tests\eLife\Search\Workflow;
+
+use ComposerLocator;
+use eLife\Search\Api\ApiValidator;
+use eLife\Search\Api\Elasticsearch\MappedElasticsearchClient;
+use eLife\Search\Workflow\BlogArticleWorkflow;
+use eLife\Search\Workflow\Workflow;
+use Mockery;
+use PHPUnit_Framework_TestCase;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Serializer\Serializer;
+use tests\eLife\Search\AsyncAssert;
+use tests\eLife\Search\ExceptionNullLogger;
+use tests\eLife\Search\HttpMocks;
+use Traversable;
+use function GuzzleHttp\json_decode;
+
+abstract class WorkflowTestCase extends PHPUnit_Framework_TestCase
+{
+    use AsyncAssert;
+    use HttpMocks;
+    use GetSerializer;
+    use GetValidator;
+
+    /**
+     * @var BlogArticleWorkflow
```

Comment:
This should be `@var Workflow`. We could then also remove the use statement for BlogArticleWorkflow